### ISSUE TITLE
1.0 - Section Background Image from Custom Field No Longer Displays (Image Element also impacted)

### ISSUE THREAD

**Issue by bmc38119** [soflyy/breakdance-bugs#317]
**Please try to clearly describe the problem.**
I have a Header template that contains a Section with a background image that is set to dynamic custom field (ACF).  In Beta 11, image displayed correctly.  In Beta 12, dynamic background image does not display.  The ACF image is set at the Page level.

I have tried the following troubleshooting steps:

- Removed current dynamic setting and replaced with new (did not work)
-- FYI - the only option is to manually enter the custom field name (the custom field is not visible or selectable in the modal displaying all available dynamic data)
- Adjusted the output of the ACF image from array to image URL to ID (none worked)

What is also interesting is after updating to Beta 12, none of my dynamic fields were displaying (except for ones where I had manually entered the custom field name).  Any that had been selected from the dynamic data dialog were not working - I had to go into each of the Global Blocks, clear out the existing and make a new selection of the same field).  So far, I have been able to correct all except for this dynamic background image in the Header template.  

**Including a screenshot of the problem is a good idea.**

Section Background Image Settings:
![image](https://user-images.githubusercontent.com/20451326/187951039-aca5ee92-528d-48dd-bfe5-ba9d4157c140.png)

ACF Custom Field: 
![image](https://user-images.githubusercontent.com/20451326/187951381-c1948863-ba62-4d71-b476-c3ed8870499e.png)

Custom Field Set with Image at Page Level:
![image](https://user-images.githubusercontent.com/20451326/187951548-372a49b4-81d0-4a39-82e2-791f4c4f508b.png)

ACF Image not displaying correctly on Page:
![image](https://user-images.githubusercontent.com/20451326/187951774-309b71d8-ae13-4e37-9c86-d0c8ca69d724.png)

I can provide credentials to site if needed.

(Or a screencast - https://www.loom.com/)

**Comment by bmc38119** [comment]
I am thinking this issue can be closed - as I can no longer replicate it.  I decided to add an image to the section and absolute position it as the background - and as soon as I did that the section background began to work properly.  I am not sure of adding this additional element to the section caused something to update related to Beta 12 - but the dynamic background image now works.

**Comment by bmc38119** [comment]
To add more to this bug report - this issue appears to affect the image element as well in the Header template.  Until the section background bug was fixed, I decided to try to absolute position an image element as a background - however, I am experiencing the same issue.  

Here is the backend showing the image inserted.
![image](https://user-images.githubusercontent.com/20451326/188173306-745023a1-9920-458b-a9d7-9478f629c646.png)

And here is the front-end result:
![image](https://user-images.githubusercontent.com/20451326/188173384-f50cfe3d-bf93-4216-98a1-2fbb8296dd27.png)

And here is the code output from inspector (image ID is being inserted as the src):
![image](https://user-images.githubusercontent.com/20451326/188173541-e277def5-10fc-4265-9eb5-7244b2064dc2.png)
